plugins/destination/postgresql/client/migrate.go

```diff
@@ -167,6 +170,50 @@ func (c *Client) dropTable(ctx context.Context, tableName string) error {
 	return nil
 }
 
+func (c *Client) migrateToCQID(ctx context.Context, tableName string, _ schema.Column) error {
+	// Steps:
+	// acquire connection
+	conn, err := c.conn.Acquire(ctx)
+	if err != nil {
+		return fmt.Errorf("failed to acquire connection: %w", err)
+	}
+	defer conn.Release()
+	// start transaction
+	tx, err := conn.BeginTx(ctx, pgx.TxOptions{})
+	if err != nil {
+		return fmt.Errorf("failed to begin transaction: %w", err)
+	}
+
+	sanitizedTableName := pgx.Identifier{tableName}.Sanitize()
+	sanitizedPKName := pgx.Identifier{getPKName(&schema.Table{Name: tableName})}.Sanitize()
```

Comment:
Can we sure this constraint name always exist in this scenario? Maybe we should ignore if it doesn't instead of failing?

It won't exist probably when we do Postgres->Postgres, but then `TableColumnChangeTypeMoveToCQOnly` should be false I think

Reply:
This function is only called for `TableColumnChangeTypeMoveToCQOnly` change... So I think that it is fine, If users are messing around with the `_cq_id` column we cannot be sure of anything... 
```
case schema.TableColumnChangeTypeMoveToCQOnly:
			return c.migrateToCQID(ctx, tableName, change.Current)
```